st2common/st2common/logging/handlers.py

```diff
@@ -37,7 +38,8 @@ def __init__(self, filename, mode='a', maxBytes=0, backupCount=0, encoding=None,
         format_values = {
             'timestamp': timestamp,
             'ts': isotime_str,
-            'pid': pid
+            'pid': pid,
+            'pack': os.path.abspath(os.path.dirname(sys.argv[2])).split('/')[-2]
```

Comment:
As we have discussed in the today's meeting, the logging is being carried for the actions with no python file  in the st2actionrunner.1062.log file in the format which was as earlier.